src/codegeneration/ModuleTransformer.js

```diff
@@ -276,8 +276,8 @@ export class ModuleTransformer extends TempVarTransformer {
     assert(this.moduleName);
     let name = tree.token.processedValue;
     // import/module {x} from './name.js' is relative to the current file.
-    let normalizedName = System.normalize(name, this.moduleName);
-    return parseExpression `System.get(${normalizedName})`;
+    let normalizedName = $traceurRuntime.normalizeModuleName(name, this.moduleName);
```

Comment:
Yes there needs to be some global reference. It can be worth wrapping all module-generated code with a function that then has that runtime module loader reference scoped, but it still needs to be a name output here.

I'm not sure I follow the question - SystemJS aims to run all module formats, including support for these types of outputs.